src/components/Personalization/createPersonalizationDetails.js

```diff
@@ -41,7 +42,12 @@ export default ({ renderDecisions, decisionScopes, event, viewCache }) => {
         scopes.push(PAGE_WIDE_SCOPE);
       }
 
-      const schemas = [HTML_CONTENT_ITEM, JSON_CONTENT_ITEM, REDIRECT_ITEM];
+      const schemas = [
+        DEFAULT_CONTENT_ITEM,
```

Comment:
I like the explicit nature of this PR.

I wish there was a way for Target Upstream to figure out which schemas to act on without a change to the Web SDK  which enable us to add more schemas in the future without a change to the SDKs..

Reply:
@jfkhoury We can make an optimization in this regard in case when content for all available schemas should be retrieved by not specifying `schemas` array at all in the request - in this case content for all available schemas will be returned implicitly from Target Upstream.
Thus, only when the page wide scope is not included in the request would we need to set the schemas explicitly to all values other than DOM_ACTION. Let me know if this would be desired and I'll open a ticket for it..

Reply:
@XDex the optimization you suggested would still require a code change when a new schema was added. Target upstream should know that only when the "__view__" scope is requested, it should return DOM_ACTION schemas. If this were the case the Web SDK would only need to be updated when new schemas needed to be handled by the SDK. Currently only redirect, and dom action schemas are handled. The rest are passed through. 

If a new pass-through schema were added, this code would need to be updated even though the SDK isn't doing anything with it.

Reply:
Right, let's just leave it as-is then for now, the schemas being always explicitly specified in the request.